src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -20,6 +22,10 @@
     <ProjectJson>unix/project.json</ProjectJson>
     <ProjectLockJson>unix/project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net46' and '$(ProjectJson)' == '' ">
+    <ProjectJson>net46/project.json</ProjectJson>
```

Comment:
I followed the pattern from the existing unix folder in this library. Although the unix project.json has special requirements in that it binds against netstandard14 instead of netstandard13 due to weird packaging issues I don't fully understand (@ericstj does).